src/lib/y2packager/dialogs/inst_product_license.rb

```diff
@@ -111,10 +112,26 @@ def language_selection
       def license_content
         MinWidth(
           80,
-          RichText(Id(:license_content), product.license(language))
+          RichText(Id(:license_content), formatted_license_text)
         )
       end
 
+      RICHTEXT_REGEXP = /<\/.*>/.freeze
+      # Return the license text
+      #
+      # It detects whether license text is richtext or not and format it
+      # accordingly.
+      #
+      # @return [String] Formatted license text
+      def formatted_license_text
+        text = product.license(language)
+        if RICHTEXT_REGEXP =~ text
+          text
+        else
+          "<pre>#{Yast::String.EscapeTags(text)}</pre>"
```

Comment:
I'd prefer native Ruby `CGI.escapeHTML` function here, we are already [using it it elsewhere](https://github.com/yast/yast-registration/blob/c603f9634520ebe2594fd3b08bb87804974c60e6/src/lib/registration/ui/migration_selection_dialog.rb#L208).

In the long term we should drop all YaST specific code and use Ruby stdlib where possible.